test/expression.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+const assert = require('assert');
```

Comment:
Can you update the test to use the same version (of syntax) of Javascript as the other test/rest of the codebase?

So no const, and no => functions.